src/topology/continuous_on.lean

```diff
@@ -428,6 +428,31 @@ lemma eventually_nhds_within_of_eventually_nhds {α : Type*} [topological_space
   ∀ᶠ x in 𝓝[s] a, p x :=
 mem_nhds_within_of_mem_nhds h
 
+lemma tendsto_nhds_within_iff_seq_tendsto {x : α} {l : filter β} {s : set α}
+  [(𝓝[s] x).is_countably_generated] (f : α → β) (hx : x ∈ s) :
+  tendsto f (𝓝[s] x) l
+    ↔ (∀ xs : ℕ → α, (∀ n, xs n ∈ s) → tendsto xs at_top (𝓝 x) → tendsto (f ∘ xs) at_top l) :=
```

Comment:
IMHO,
* this lemma should go to `topology.continuous_on`;
* there should be a version `tendsto_inf_principal_iff_seq_tendsto` near `filter.tendsto_iff_seq_tendsto`;
* the proof should lift `x` to `s`, then use `tendsto_iff_seq_tendsto` applied to `comap (coe : s → α) l`;
* there could be a version of `frechet_urysohns_space`s in this file if you need it for some reason.

Reply:
Thanks for the suggestions! I'll implement those changes.
About the first one though: this lemma is already in that file?

Reply:
Do you mean it should go to `topology.sequences`?

Reply:
Should `f`, `x`, and `s` be implicit?